Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/RevitSendBinding.cs

```diff
@@ -346,15 +346,33 @@ private async Task RunExpirationChecks()
     }
 
     var objUniqueIds = new List<string>();
+    var changedIds = ChangedObjectIds.Keys.ToList();
```

Comment:
Perf: Do we need to make another collection here?

Reply:
i am adding to it below, so i kinda need it in there